includes/multi-currency/Compatibility/WooCommerceSubscriptions.php

```diff
@@ -159,12 +159,12 @@ public function override_selected_currency( $return ) {
 		$switch_cart_items = $this->get_subscription_switch_cart_items();
 		if ( 0 < count( $switch_cart_items ) ) {
 			$switch_cart_item = array_shift( $switch_cart_items );
-			return get_post_meta( $switch_cart_item['subscription_switch']['subscription_id'], '_order_currency', true );
+			return $switch_cart_item['subscription_switch']['subscription_id']->get_currency();
```

Comment:
Similarly for these lines as well, however since we're using the subscription ID we should get a subscription object using:
`wcs_get_subscription( $subscription_resubscribe['subscription_resubscribe']['subscription_id'] )`

Then call `get_currency()` on that object 😃